tests/utils/kudo/kudo.go

```diff
@@ -411,7 +411,7 @@ func UpgradeOperator(
 func UninstallOperator(
 	operatorName string, namespaceName string, instanceName string,
 ) error {
-	uninstallScript := "../../scripts/uninstall_operator.sh"
+	uninstallScript := "../../../scripts/uninstall_operator.sh"
```

Comment:
This is required because the suites moved one directory deeper? 

Maybe we should pass in the scripts directory as an absolute path at some point and not rely on relative paths. Seems to be ok for now though.

Reply:
Would be ok for now for me too. Is there a way to make this relative to the actual `kudo.go` file instead, though?

Reply:
@ANeumann82 right, later we can do something like this: https://github.com/mesosphere/kudo-kafka-operator/pull/6/files#diff-dc3495b5db55989eec09ceab2776f370R29
Set a `REPO_ROOT` env var and create abs paths using that

Reply:
It's already available to be passed as an environment variable fwiw: https://github.com/mesosphere/kudo-cassandra-operator/blob/7a143804c906b3f8c5fda40f359ab297506c35ff/run-tests.sh#L7